dynamo/preprocessing/Preprocessor.py

```diff
@@ -678,15 +672,20 @@ def preprocess_adata_pearson_residuals(
                 would be inferred from the data. Defaults to None.
         """
 
-        temp_logger = LoggerManager.gen_logger("preprocessor-sctransform")
+        temp_logger = LoggerManager.gen_logger("Preprocessor-pearson residual")
         temp_logger.log_time()
         self.standardize_adata(adata, tkey, experiment_type)
 
```

Comment:
Need confirmation here. I think Pearson residuals also need filters.

Reply:
Now all of recipes have below 3 basic steps.
```
self.standardize_adata(adata, tkey, experiment_type)
self._filter_cells_by_outliers(adata)
self._filter_genes_by_outliers(adata)
```